<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changePasswordDialog</class>
 <widget class="QDialog" name="changePasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>351</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="change_newpass_label">
      <property name="text">
       <string>New Password</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="changepass_newpass_lineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="change_confirmpass_label">
      <property name="text">
       <string>Confirm Password</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="changepass_confirmpass_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="changepass_change_pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changepass_cancel_pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
